charts/cluster-scanner/tests/existing-secret_test.yaml

```
@@ -21,7 +21,7 @@ tests:
       global.sysdig.accessKeySecret: "secret"
     asserts:
       - equal:
-          path: spec.template.spec.containers[0].env[35]
+          path: spec.template.spec.containers[0].env[39]
```

Instead of using the positional notation, I'd recommend using something like:

```suggestion
          path: spec.template.spec.containers[0].env[?(@.name == "SYSDIG_ACCESS_KEY")]
```

This applies also to the other assertion at line 32